File: python/Food.py
from Amenity import *

class Food(Amenity):
    def __init__(self):
        super().__init__()
        self.price = 200
    
    def getDescription(self):
        return "Food amenity at 200 baht per time"

class FoodPlus(Amenity):
    def __init__(self):
        super().__init__()
        self.price = 300
    
    def getDescription(self):
        return "FoodPlus amenity at 300 baht per time"

File: Food.java
public class Food extends Amenity {

    public Food() {
        super(200);
    }

    public Food(float price) {
        super(price);
    }

    public String getDescription() {
        return "Food amenity at " + this.getPrice() + "Baht per time" ;
    }

}
